### PR TITLE
theme: add missing links

### LINKED ISSUE #2703: Missing links
When comparing the detailed view of the same record on legacy and labs, we see some links missing
Eg: links to OSTI and ADS are missing from Labs:  https://labs.inspirehep.net/literature/212819 

osti2

Whereas we see them in Legacy: https://inspirehep.net/record/212819
osti

### PR DESCRIPTION
**Pull request by Dinika**
Signed-off-by: Dinika <dinika.saxena@cern.ch>

## Description
I have added a link for the OSTI Institute. This fixes the problem for the record
mentioned in issue #2703. But links for other institutes might still be missing.

## Related Issue
Fixes #2703 

## Checklist:

- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [x] I linked the related issue(s) in the corresponding commit logs.
- [x] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [x] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl [hunk 1]
```diff
@@ -136,6 +136,7 @@
             </div>
             <div class="modal-body">
             <h4>INSPIRE keywords</h4>
+
```
Comment (jacquerie) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: Watch out! The stuff above this line doesn't belong to this PR.

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl [hunk 2]
```diff
@@ -179,9 +179,9 @@
     {% endfor %}
   {% endif %}
 
-  {% if record.get('external_system_numbers') %}
-    {% for system_number in record.get('external_system_numbers') if system_number.get('obsolete') == False and system_number.get('institute', '') | is_institute %}
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: @michamos is there an equivalent now of `obsolete` or `institute` that should be skipped when displaying external URLs?

Comment (michamos) [reply inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: not as explicit flags. The logic is now that the first one with a given schema is to be displayed, the others are discarded. So
```python
{
    'external_system_numbers': [
        {
            'schema': 'foo',
            'value': 'val1',
        },
        {
            'schema': 'bar',
            'value': 'val2',
        },
        {
            'schema': 'foo',
            'value': 'val3',
        }
    ]
}
```
should display the first two elements only, because `foo` is repeated.

It's probably good to write a reusable getter that returns the filtered list to be displayed, and while you are at it refactor this to compute the links in python code instead of in the templates.

Comment (jmartinm) [reply inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: @Dinika in this file https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/wrappers.py#L43 you can create a new property for that class to get the external_system_identifiers.

This property will filter the list keeping the order but removing any duplicated schema.

Then in the template you can iterate over it with `record. external_system_identifiers`

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl (hunk 1 above)
Comment (jmartinm) [reply inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: @Dinika please look at this. The above changes are already in `master` branch, so rebase and they should go away.

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl (hunk 2 above)
Comment (jmartinm) [reply inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_HTML_detailed_macros.tpl]: This last comment is still missing in order to avoid showing duplicates.

**Comment by jmartinm** [comment]
@Dinika is this PR still relevant? or newer PRs have already implemented this functionality and can be closed?

**Comment by Dinika** [comment]
@jmartinm I forgot to close this PR. PR #2847 now fixes this issue.
